src/clj/clj_diff/miller.clj

```diff
@@ -240,20 +250,26 @@
   string-dispatch)
 
 (defmethod diff :default
-  [a b opts]
-  (seq-diff opts a b))
+  ([a b]
+   (diff a b {}))
+  ([a b opts]
+   (seq-diff a b opts)))
 
 (defmethod diff :string
-  [a b opts]
-  (opt/diff a b (partial seq-diff opts)))
+  ([a b]
+   (diff a b {}))
+  ([a b opts]
+   (opt/diff a b (fn [a b] (seq-diff a b opts)))))
 
 (defn seq-edit-dist
-  [a b opts]
-  (let [[a b] (vectorize a b)
-        [[p & more] a* b*] (order->ses a b opts)]
-    (if (neg? p)
-      p
-      (+ (* 2 p) (- (count a*) (count b*))))))
+  ([a b]
+   (seq-edit-dist a b {}))
+  ([a b opts]
+   (let [[a b] (vectorize a b)
+         [[p & more] a* b*] (order->ses a b opts)]
+     (if (neg? p)
+       p
+       (+ (* 2 p) (- (count a*) (count b*)))))))
 
 (defmulti edit-distance string-dispatch)
```

Comment:
As mentioned in https://github.com/droitfintech/clj-diff/pull/7#discussion_r946738839, `edit-distance`'s args should also be adjusted to avoid a breaking change.

Reply:
ok